realtime/utils.js

```diff
@@ -445,8 +446,19 @@ function attachAspectSubject(sample, useSampleStore, subjectModel,
     let sub = response[1];
     asp = asp.get ? asp.get() : asp;
     sub = sub.get ? sub.get() : sub;
+
+    // if asp or sub from redis, writers field == [] instead of undefined
+    if (asp.writers === '[]') {
+      delete asp.writers;
+    }
+
+    if (sub.writers === '[]') {
+      delete sub.writers;
+    }
```

Comment:
I think we can delete the aspect and subject writers no matter what.

Reply:
What if the aspect.writers or subject.writers = ['user1', 'user2', ...]? Do we still want to delete the aspect and subject writers?